src/content/docs/fr/tutorial/6-islands/3.mdx

```diff
@@ -0,0 +1,74 @@
+---
+type: tutorial
+title: Félicitations !
+description: >-
+  Tutoriel : Créez votre premier blog Astro —
+
+  Découvrez la version finale de votre projet et découvrez ce qui vous attend avec Astro !
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Box from '~/components/tutorial/Box.astro';
+import Checklist from '~/components/Checklist.astro';
+import CompletionConfetti from '~/components/tutorial/CompletionConfetti.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
```

Comment:
```suggestion
import PreCheck from '~/components/tutorial/PreCheck.astro';
import Button from '~/components/Button.astro';
```

The preview is not up yet but I think this import is needed for the share buttons.

Reply:
Right i didn't see it